apps/src/templates/teacherDashboard/PrintCertificates.jsx

```diff
@@ -3,9 +3,7 @@ import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import $ from 'jquery';
 import i18n from '@cdo/locale';
-import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
 import style from './print-certificates.module.scss';
-import experiments from '@cdo/apps/util/experiments';
 import RailsAuthenticityToken from '@cdo/apps/lib/util/RailsAuthenticityToken';
 
```

Comment:
I'm seeing a todo in this file: "// TODO(dave): remove assignmentName prop after launching studioCertificate experiment" –– are you doing this in a separate PR?

Reply:
yep! these are the follow-up PRs in the work plan:
* [#48203](https://github.com/code-dot-org/code-dot-org/pull/48203) clean up client code after removing experiment flag
* [#48211](https://github.com/code-dot-org/code-dot-org/pull/48211) replace client-side redirects with server-side redirects, and delete the pegasus congrats pages

this is removed here: https://github.com/code-dot-org/code-dot-org/pull/48203/files#diff-f7a08f8b568870f8f9703233ff14f5e217037462084f7246ed41142d14c5b87dL12

Reply:
Ah sounds great! Thanks